python_scripts/ensemble_bagging.py

```diff
@@ -263,20 +263,11 @@ def bootstrap_sample(data, target):
 # Let us compare the based model predictions with their average:
 
 # %%
-import warnings
-
-with warnings.catch_warnings():
-    # ignore scikit-learn warning when accessing bagged estimators
-    warnings.filterwarnings(
-        "ignore",
-        message="X has feature names, but DecisionTreeRegressor was fitted without feature names",
-    )
-
-    for tree_idx, tree in enumerate(bagged_trees.estimators_):
-        label = "Predictions of individual trees" if tree_idx == 0 else None
-        tree_predictions = tree.predict(data_test)
-        plt.plot(data_test, tree_predictions, linestyle="--", alpha=0.1,
-                 color="tab:blue", label=label)
+for tree_idx, tree in enumerate(bagged_trees.estimators_):
+    label = "Predictions of individual trees" if tree_idx == 0 else None
```

Comment:
```suggestion
    label = "Predictions of individual trees" if tree_idx == 0 else None
    # we convert `data_test` into a NumPy array to avoid a warning raised in scikit-learn
```